tox.ini

```diff
@@ -0,0 +1,157 @@
+[tox]
```

Comment:
is it possible to don't use tox?

the plone framework team choose to don't use tox for plone core, which don't mean we should follow them in one addon

but I also had some bad experience with `tox` at my current work because it makes the test workflow really slow